posthog/migrations/0490_dashboard_variables.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.15 on 2024-10-04 15:49
```

Comment:
Probably should break this out and have it run as a separate PR. Keep the dashboard variable logic isolated incase we need to revert

Reply:
Done!